badges/tests/test_recalculation_batching.py

```
@@ -0,0 +1,255 @@
+    assert batched.call_count == 1
+    assert batched.call_args.args[0] == member.pk
+
+
+def test_the_command_does_not_repeat_what_the_run_already_did():
+    """The final pass covers what the run left owing, not what it finished.
+
+    Both call sites used to recalculate every member in ``changed``, on the belief
+    that the signal had done the removals - which it had, and then they did them
+    again. ``outstanding`` is what makes that division of labour real.
+    """
+    member, author = _member_with_commits("outstanding@example.com", 2)
```

Ruff (0.16.1)

[warning] Unpacked variable `member` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)